lib/common/formats.js

```diff
@@ -110,6 +97,53 @@ module.exports = {
       .value().join('\n');
   },
 
+  /**
+   * Creates a LESS file with variable definitions based on the style dictionary
+   *
+   * @memberof Formats
+   * @kind member
+   * @example
+   * ```less
+   * @color-background-base: #f0f0f0;
+   * @color-background-alt: #eeeeee;
+   * ```
+   */
+  'less/variables': function(dictionary) {
+    var to_ret;
+    return fileHeader() + _.map(dictionary.allProperties, function (prop) {
+      to_ret = '@' + prop.name + ': ' + (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value) + ';';
```

Comment:
I noticed there's a little repetition here and with the sass (similar to CSS as well); We could create a method that receives all these props and the prefix and should automatically create the array with the correct value and use the same method across these transforms; Thoughts?

Reply:
Yeah, I'll update it